components/src/core/hero/hero.component.ts

```diff
@@ -45,13 +51,21 @@ import { requireInput } from '../../utils/utils';
     },
 })
 export class HeroComponent implements OnChanges, AfterViewInit, AfterContentChecked {
+    /** Deprecated* Color of the hero icon */
     @Input() color: string;
-    @Input() label: string;
-    @Input() value: string;
-    @Input() units: string;
+    /** Deprecated* Color of the hero background */
```

Comment:
Why are these deprecated? There is a specific syntax for deprecated properties using '@ deprecated'

Reply:
I marked these as deprecated because we had talked about removing these properties months ago & they aren't even documented in our READMEs.